src/main/resources/reference.conf

```diff
@@ -59,24 +57,20 @@ batchedFetches {
   #   said operations, and be compared to the load handling capabilities
   #   of the Tezos node itself
 
-  #Used when getting accounts from tezos
-  accountConcurrencyLevel = 5
+  #Used to set the concurrency level of calls to the remote node for new blocks
+  blockFetchConcurrencyLevel = 200
```

Comment:
That's a big number. Won't it slow down processing?

Reply:
It's always been around that number, processing is lazily done as records arrive, via fs2 stream.

Right now we should already hit tezos nodes with that size of open requests for any kind of data

Reply:
In any case I'm curious to understand what's on your mind?

What can cause slow-downs if we get that many entries concurrently?